source/_reference/methods.coffee

```diff
@@ -511,25 +539,50 @@ module.exports = exports =
       httpOnly: true
       http: "GET /events/{id}/{fileId}[/{fileName}]"
       description: """
-                   Gets the attached file. Accepts an arbitrary filename path suffix (ignored) for easier link creation.
+                   Gets the attached file. Accepts an arbitrary filename path suffix (ignored) for easier link readability.
+                   For this function using the `auth` query parameter is not accepted. You can either use the [access token](##{dataStructure.getDocId("access")}) in the `Authorization` header or provide the `readToken` query parameter.
                    """
       params:
         properties: [
+          key: "id"
+          type: "[identifier](##{dataStructure.getDocId("identifier")})"
+          http:
+            text: "set in request path"
+          description: """
+                       The id of the event.
+                       """
+        ,
+          key: "fileId"
+          type: "[identifier](##{dataStructure.getDocId("identifier")})"
+          http:
+            text: "set in request path"
+          description: """
+                       The id of the attached file.
+                       """
+        ,
+          key: "fileName"
+          type: "string"
+          optional: true
+          http:
+            text: "set in request path"
+          description: """
+                       The name of the file.
+                       """
+        ,
           key: "readToken"
           type: "string"
+          optional: true
```

Comment:
Not fan of optional here, we have to provide it even if we can choose how (headers/query).

Reply:
What about writing something like for the `mergeEventsWithParent` param in [streams.delete](http://api.pryv.com/reference/#delete-stream): start the description part with`Required if ...`?

Reply:
No, you don't need to provide it if you provide an access token in the authorization header, see Trello card.